GOVERANCE.md

```diff
@@ -0,0 +1,200 @@
+# Open Cluster Management Project Governance
+
+## Table of Contents
+
+- [Values](#values)
+- [Project Governance](#project-governance)
+  - [Adding New Subprojects](#adding-new-subprojects)
+  - [Removing Projects](#removing-projects)
+- [Steering Committee](#steering-committee)
+  - [Duties](#duties)
+  - [Members](#members)
+  - [Elections](#elections)
+  - [Decision Making](#decision-making)
+- [Code of Conduct](#code-of-conduct)
+
+# Values
+
+Open Cluster Management and its leadership embrace the following values:
+
+- Openness: Open Cluster Management is an open source community driven project.
+- Distributed design: Distributed asynchronous ownership, collaboration, communication and decision making are the cornerstones of our world-wide community.
+- Fairness: Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles.
+- Diversity and inclusion: Everyone are welcome to participate in the Open Cluster Management project regardless of gender, gender identity, sexual orientation, disability, race, ethnicity, age, religion or economic status.
+
+# Project Governance
+
+The project consists of the following Subprojects:
```

Comment:
How do we define subprojects here? Based on the CNCF incubation checkbox:

> If the project has subprojects: subproject leadership, contribution, maturity status documented, including add/remove process. 

So, Registration, Work, Scheduling, Operator might not be the Subprojects, and we should list [addon-framework](https://github.com/open-cluster-management-io/addon-framework), [sdk-go](https://github.com/open-cluster-management-io/sdk-go/blob/main/OWNERS) and [clusteradm](https://github.com/open-cluster-management-io/clusteradm) here. 

What do you think?

Reply:
And refer to the example in https://github.com/cncf/toc/blob/main/projects/cubefs/cubefs-graduation-dd.md?plain=1#L213, the below governance template should be added to each subproject:

Governance
- _< subproject-name >_ is a sub-project of the OCM main project, complying with the rules of OCM main projects.
- _< subproject-name >_ does not have independent leadership, adopting the same leadership strategy as the OCM main project.
- Contributions to _< subproject-name >_ are equivalent to contributions to the main project and can be used as a basis for community role promotion.

Reply:
That's a good question, we should discuss this in the incubation meeting.